### PR TITLE
clean up to fix library error

### PR DESCRIPTION
**Pull request by xlqian**
# Issue

We checked out the latest version (master) to compile inside of our project. Some errors occurred when compiling due to the "bad" include in some headers. Here is the fix 

## Tasklist

 - [x] Update the [changelog](CHANGELOG.md)

### CODE REVIEW

File: valhalla/mjolnir/osmway.h [hunk 1]
```diff
@@ -6,13 +6,15 @@
 #include <string>
 #include <vector>
 
-#include "mjolnir/osmpronunciation.h"
```
Comment (kevinkreiser) [inline valhalla/mjolnir/osmway.h:9]: couldnt the fix have been as simple as <valhalla/mjolnir/osmpronunciation.h> here, and then don't do any of the other changes?

Comment (xlqian) [reply valhalla/mjolnir/osmway.h:9]: unluckily No, that was the first thing I did LOL

Comment (xlqian) [reply valhalla/mjolnir/osmway.h:9]: oops, that idea seems should work.. I have done something wrong certainly...

**Comment by kevinkreiser** [comment]
for some odd reason the build is breaking recently on the edge filter tests in release mode. we might want to look into that. for now ive just rerun the build but its happened a few times now so there is definitely something up. not related to this change, this looks good to me

**Comment by xlqian** [comment]
I was just trying to find out what has broken the CI in my PR :smiling_face_with_tear:

**Review by kevinkreiser — APPROVED** [review]

**Pull request merged**